### PR TITLE
Release announcement 2.2.0

### PR DESCRIPTION
**Pull request by gassmoeller**
Suggestions for changes to the release announcement. I started from my own version of the announcement so the text is slightly different. I dont particularly care about the text, but would like to keep the author names (we discussed this a while ago at one of the developer meetings).

### CODE REVIEW

**Comment by gassmoeller** [comment]
relates to geodynamics/aspect#3567

File: doc/release-tasklist [hunk 1]
```diff
@@ -147,66 +147,105 @@ and
 
 This release includes the following significant changes:
 
-- New: ASPECT now contains a new matrix-free Stokes solver that uses geometric
-  multigrid with substancially better performance and lower memory
-  requirements.
+- New: There is a new matrix-free Stokes solver which uses geometric multigrid for 
+  preconditioning the A-block. This method is significantly faster than
```
Comment (tjhei) [inline doc/release-tasklist:151]: remove "for preconditioning the A-block". We do a lot of other stuff.

File: doc/release-tasklist [hunk 2]
```diff
@@ -147,66 +147,105 @@ and
...
+  the default algebraic multigrid preconditioner and uses less memory. Free
+  surface, and melt transport are not yet implemented.
```
Comment (tjhei) [inline doc/release-tasklist:153]:
```suggestion
  surface and melt transport are not yet implemented.
```

File: doc/release-tasklist [hunk 3]
```diff
@@ -147,66 +147,105 @@ and
...
+  (Conrad Clevenger, Timo Heister)
 
-- New: There is now a new approximation for the compressible convection models
-  that is called 'projected density field'.
+- New: There is now a new approximation for the compressible convection
+  models that is called 'projected density field'.
+  (Rene Gassmoeller, Juliane Dannberg, Timo Heister, Wolfgang Bangerth)
 
-- Changed: The Geodynamic World Builder has been updated to version 0.3.0.
+- Changed: The Geodynamic World Builder has been updated to version 0.3.0
```
Comment (tjhei) [inline doc/release-tasklist:160]:
```suggestion
- Changed: The Geodynamic World Builder has been updated to version 0.3.0.
```

File: doc/release-tasklist [hunk 4]
```diff
@@ -147,66 +147,105 @@ and
...
-  using SUPG has been added.
+- New: The IDR(s) Krylov method can now be used for solving the Stokes system,
+  when using the matrix-free GMG solver. It has a short term recurrence and can
+  greatly reduce the memory requirement compared to GMRES.
+  (Thomas C. Clevenger, Timo Heister)
 
-- Changed: The entropy viscosity method for stabilizing the advection
-  equations for temperature and composition was substantially improved.
+- New: There is a new, alternative stabilization method for the advection equation 
+  called SUPG. This is preliminary work.
```
Comment (tjhei) [inline doc/release-tasklist:172]:
```suggestion
  called SUPG. 
```

File: doc/release-tasklist [hunk 5]
```diff
@@ -147,66 +147,105 @@ and
...
 
-- New benchmarks: entropy equation, viscoelastic cantilever, bouyancy-driven
-  viscoelastic plate, Advection in annulus, Slab detachment benchmark, several
-  advection benchmarks, rigid shear
+- Changed: ASPECT now requires deal.II version 9.0.0 or newer.
+  (Timo Heister, Rene Gassmoeller)
 
-- New: A new, alternative advection stabilization scheme to Entropy Viscosity
-  using SUPG has been added.
+- New: The IDR(s) Krylov method can now be used for solving the Stokes system,
```
Comment (tjhei) [inline doc/release-tasklist:166]: I would remove this. This is part of the first GMG entry.

**Comment by gassmoeller** [comment]
done

**Pull request merged**